### PR TITLE
Remove method to publish to Debian feeds.

### PR DESCRIPTION
**Pull request by ghost**
**Follow-up actions:**
Remove the variables:
"REPO_SERVER"
"REPO_PASS"
"REPO_USER"
"debian8x64RepositoryId"
"ubuntu1404x64RepositoryId"
"ubuntu1604x64RepositoryId"
from CLI:master VSO build definitions.

### CODE REVIEW

**Review by livarcocc — APPROVED** [review]

File: build/publish/PublishDebian.targets [hunk 1]
```diff
@@ -1,23 +0,0 @@
...
-  </PropertyGroup>
-
-  <Target Name="PublishDebFilesToDebianRepo"
-          DependsOnTargets="SetupDebProps;"
-          Condition=" '$(IsBuildingAndPublishingAllLinuxDistrosNativeInstallers)' == 'true' AND '$(SkipPublishToDebianRepo)' != 'true' AND '$(IslinuxPortable)' != 'true' " >
-    <Error Condition="'$(REPO_USER)' == ''" Text="REPO_USER must be set as a MsBuild Property variable for debian publishing." />
-    <Error Condition="'$(REPO_PASS)' == ''" Text="REPO_PASS must be set as a MsBuild Property variable for debian publishing." />
-    <Error Condition="'$(REPO_SERVER)' == ''" Text="REPO_SERVER must be set as a MsBuild Property variable for debian publishing." />
-
-    <UploadToLinuxPackageRepository Condition=" '%(LinuxDistrosNeedNativeInstaller.RepositoryId)' != '' "
```
Comment (eerhardt) [inline build/publish/PublishDebian.targets]: Can we remove the `UploadToLinuxPackageRepository` task code as well?

File: build/LinuxDistrosNativeInstaller.props [hunk 2]
```diff
@@ -3,13 +3,11 @@
     <LinuxDistrosNeedNativeInstaller Include="debian.8-x64">
       <DockerFolder>debian</DockerFolder>
       <InstallerExtension>deb</InstallerExtension>
-      <RepositoryId>$(debian8x64RepositoryId)</RepositoryId>
```
Comment (eerhardt) [inline build/LinuxDistrosNativeInstaller.props]: These variables are specified in the build definition, right?  I assume they will be removed once this is merged.

Comment (johnbeisner) [reply build/LinuxDistrosNativeInstaller.props]: Yes

File: build/publish/PublishDebian.targets (hunk 1 above)
Comment (johnbeisner) [reply build/publish/PublishDebian.targets]: Done...

**Review by livarcocc — APPROVED** [review]
Yes!

**Review by eerhardt — APPROVED** [review]
Thanks @johnbeisner

**Pull request merged**